src/frontend/src/styles/main.css

```diff
@@ -794,6 +797,133 @@ a:hover,
   box-shadow: 0 0 0 4px rgba(255, 255, 255, 0.4);
 }
 
+/**
+ *  irregularity component
+ */
+.c-irregularity {
+  display: flex;
+  align-items: stretch;
+  gap: calc(var(--rs-card-bezel-x) * 0.25);
+
+  --state-color: var(--rc-warningLight);
```

Comment:
what is `--state-color`? "state" suggests that it might change but it seems to be set once and for all